src/main/java/frc/robot/commands/ScoringCommands.java

```diff
@@ -15,18 +16,16 @@ public static Command scoreAmp(Elevator elevator, Indexer indexer) {
   }
 
   public static Command shootSpeaker(
-      Elevator elevator, Indexer indexer, Shooter shooter) { // TODO: Add pivot
+      Elevator elevator, Indexer indexer, Shooter shooter, ShooterPivot shooterPivot) {
     return Commands.sequence(
         elevator.setPositionCommand(ElevatorConstants.SHOOTING_POSITION),
         Commands.parallel(
-            indexer.alignForShot()
-            // TODO: Add spinup shooter command
-            // TODO: Add change shooter angle command
-            ),
+            indexer.alignForShot(),
+            shooter.SetVelocityCommand(
+                Constants.ShooterConstants.TOP_MOTOR_SETPOINT_SPEAKER,
+                Constants.ShooterConstants.TOP_MOTOR_SETPOINT_SPEAKER),
+            shooterPivot.setAngleCommand(tempval)),
```

Comment:
@ConradSkelly Code does not compile since tempval is not a symbol. Change this to null so the code can compile and add a TODO to indicate what information needs to go in here.